svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -865,7 +865,7 @@ if not svo.systemloaded then
     if not conf.curemethod or conf.curemethod == 'auto' then
       conf.curemethod = 'conconly'
     end
-  end)
+  end, 'post emit systemstart')
```

Comment:
This should be 'adjust cure method' - otherwise people will get the wrong idea when copy/pasting and call something else 'post emit systemstart'